telemetry/src/main/java/com/newrelic/telemetry/TelemetryClient.java

```diff
@@ -169,6 +169,10 @@ private void scheduleBatchSend(
 
   private void sendWithErrorHandling(
       BatchSender batchSender, TelemetryBatch<? extends Telemetry> batch, Backoff backoff) {
+    if (executor.isShutdown()) {
+      LOG.info("TelemetryClient background Executor is shutdown. Batch not sent", batch.toString());
```

Comment:
I think if we just throw the toString into the log message, it'll end up being potentially huge log line. How about just logging the number of items in the batch, and the type of batch it is (`batch.getClass().getSimpleName()`)?